config/config.go

```diff
@@ -44,6 +44,7 @@ type StorageProviderConfig struct {
 	RateLimiter        *localhttp.RateLimiterConfig
 	DiscontinueCfg     *stopserving.DiscontinueConfig
 	MetadataCfg        *metadata.MetadataConfig
+	BandwidthLimiter   *localhttp.BandWidthLimiterConfig
```

Comment:
Rename `BandWidthLimiterConfig ` to `BandwidthLimiterConfig `.

Reply:
fixed